bin/openapi2postmanv2.js

```diff
@@ -1,22 +1,57 @@
 #!/usr/bin/env node
-var program = require('commander'),
+var _ = require('lodash'),
+  program = require('commander'),
   Converter = require('../index.js'),
   fs = require('fs'),
   path = require('path'),
+  availableOptions = require('../lib/options').getOptions('use', { usage: ['CONVERSION'] }),
   inputFile,
   outputFile,
   prettyPrintFlag,
+  configFile,
+  definedOptions,
   testFlag,
   swaggerInput,
   swaggerData;
 
+/**
+ * Parses comma separated options mentioned in command args and generates JSON object
+ *
+ * @param {String} value - User defined options value
+ * @returns {Object} - Parsed option in format of JSON object
+ */
+function parseOptions (value) {
+  let definedOptions = value.split(','),
+    parsedOptions = {};
+
+  _.forEach(definedOptions, (definedOption) => {
+    let option = definedOption.split('=');
+
+    if (option.length === 2 && _.includes(_.keys(availableOptions), option[0])) {
+      try {
+        // parse parsable data types (e.g. boolean, integer etc)
+        parsedOptions[option[0]] = JSON.parse(option[1]);
+      }
+      catch (e) {
+        // treat value as string if can not be parsed
+        parsedOptions[option[0]] = option[1];
+      }
+    }
+    else {
+      console.warn('\x1b[33m%s\x1b[0m', 'Warning: Invalid defined option ', option[0]);
```

Comment:
`definedOptions` terminology seems confusing to me. These options are something the user supplies through the cli right?
We can probably omit the word `defined` and just have: `Warning: Invalid option ` or maybe `Warning: Invalid option supplied `?

Reply:
Hmm, `Warning: Invalid option supplied` looks good. 👍